src/actions/noting/toggle-note-classes.js

```diff
@@ -21,7 +21,7 @@ module.exports = function toggleNoteClasses(notes, className) {
     action = toggleClass;
   } else {
     //if we have more than one note then we want them all to share state
-    var state = collapseState.get();
+    var state = notes.every(noteSegment => hasClass(noteSegment.vNode, className));
```

Comment:
No `collapseState` is used when you collapse/expand _all_ notes. As far as I could tell, it was incorrectly used here.